### PR TITLE
Add available commands documentation.

### PR DESCRIPTION
**Pull request by BernardoSilva**
Work in progress to add commands documentation...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,62 @@ See the commands available by running
...
+- [Up](#up-command)
+- [Down](#down-command)
+- [ToggleMigration](#togglemigration-command)
+
+## GenerateMigration Command
+
+Generate a base migration backbone.
+
+```sh
+app/console mograte:generateMigration <migration-name>
```
Comment (amyboyd) [inline README.md]: Isn't it `generate-migration`?

Comment (amyboyd) [inline README.md]: `mongrate` spelled wrong here and other places.

File: README.md [hunk 2]
```diff
@@ -41,6 +41,62 @@ See the commands available by running
...
+```sh
+app/console mograte:down <migration-name>
+```
+
+## ToggleMigration Command
+
+Toggle a migration `up` or `down`.
+
+```sh
+app/console mograte:toggleMigration <migration-name>
```
Comment (amyboyd) [inline README.md]: Isn't it `toggle-migration`?

Comment (BernardoSilva) [inline README.md]: Its just `toggle`. :) 

``` sh
Available commands for the "mongrate" namespace:
  mongrate:down                 
  mongrate:generate-migration   Create a migration class
  mongrate:list-migrations      List available migrations.
  mongrate:toggle               Toggle a migration up or down. Useful when writing your migration.
  mongrate:up  
```

**Pull request merged**